enforce-rules.js

```diff
@@ -24,6 +24,10 @@ function rule0Evaluation(message, queue) {
       let hQueue = queue.filter(mBlock => {
          return mBlock.msg.speaker == "Human";
       });
+      let aQueue = queue.filter(mBlock => {
+         return mBlock.msg.speaker == "Watson" || mBlock.msg.speaker == "Celia";
```

Comment:
Easier to do these sorts of checks by using the [`Array.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) operator as it's easier to add more options, or whatever:

```suggestion
         return ['Watson', 'Celia'].includes(mBlock.msg.speaker);
```